index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Project One</title>
+</head>
+<body>
+    <header>
+        <h1>Project One Title</h1>
+    </header>
+    <div>
+        <h2>Do You Want to Do Something?</h2>
+        <label>Activity Search</label>
+        <input type="text" name="activity" placeholder="Go for a walk">
+    </div>
+    <div>
+        <label>Participants</label>
+        <select name="participants">
+            <option value="any">any</option>
```

Comment:
This might be more valuable as an input, so you can choose to have a wider variance in numbers,